### ISSUE TITLE
[ERROR] Group 'egh_gripper' is not a chain` when launching sim

### ISSUE THREAD

**Issue by scploeger** [RobotnikAutomation/rbkairos_sim#13]
When launching the sim with the following command, 
`roslaunch rbkairos_sim_bringup rbkairos_complete.launch moveit_movegroup_a:=true`
...the following error is shown:
`[ERROR] [1637254288.021776843]: Group 'egh_gripper' is not a chain`
`[ERROR] [1637254288.021797742]: Kinematics solver of type 'kdl_kinematics_plugin/KDLKinematicsPlugin' could not be initialized for group 'egh_gripper'`
`[ERROR] [1637254288.021820338]: Kinematics solver could not be instantiated for joint group egh_gripper.`

This presents future errors down the line when trying to write custom packages that use moveit.

I am using ROS Melodic on Ubuntu 18.04 with the rbkairos_sim package and dependencies installed from source.

Thanks in advance for any help!

-Spencer

**Comment by alex-arnal** [comment]
Hi, Spencer

I have updated the moveit package so the error should not appear now.  Please, try downloading again the packages and tell me if changes fix the issues that you are facing.

Álex.

**Comment by scploeger** [comment]
Thanks @alex-arnal! All seems to be working now.

Can you please explain what these repo version updates you made do and why you had to make them? Where did you get the updated version numbers from? Appreciate it,

Spencer

**Comment by scploeger** [comment]
Hi @alex-arnal sorry for confusion, for me it it still not fixed.

What I tried:
`git stash`'d any changes I had made in repo
`git pull` to get newest changes
ran `catkin_make`
launched sim with `roslaunch rbkairos_sim_bringup rbkairos_complete.launch moveit_movegroup_a:=true`

I still get following errors:

`[ERROR] [1637780880.547363164, 3.065000000]: Group 'egh_gripper' is not a chain`
`[ERROR] [1637780880.547383718, 3.065000000]: Kinematics solver of type 'kdl_kinematics_plugin/KDLKinematicsPlugin' could not be initialized for group 'egh_gripper'`
`[ERROR] [1637780880.547414935, 3.065000000]: Kinematics solver could not be instantiated for joint group egh_gripper.
`

I now also get the following warnings which did not appear before. Seems to be an issue with the plugins for laser scanner, now the robot cannot localize in rviz.

`[ WARN] [1637780898.103095989, 20.014000000]: Timed out waiting for transform from robot_base_footprint to robot_map to become available before running costmap, tf error: canTransform: target_frame robot_map does not exist.. canTransform returned after 0.1 timeout was 0.1.`
`[ WARN] [1637780898.224060034, 20.131000000]: No laser scan received (and thus no pose updates have been published) for 20.131000 seconds.  Verify that data is being published on the /robot/front_laser/scan topic.
`

**Comment by alex-arnal** [comment]
Hi Spencer,

Last changes does not update the content of this package itself, but the reference to dependencies (referenced in repos folder of this repository). The quickest way to update the simulation is following the readme to download the newest version of each package. In this case, if you only pull this repository, the simulation should work exactly as before because I just updated the references to the other repositoroes.

The biggest update has been done in rbkairos_common, where I have updated the moveit package that you were using to a newer version and I changed the gripper defining it as a set of joints instead of a chain.

Álex
